### DISCUSSION TITLE
Install.sh Error

### DISCUSSION THREAD

**Discussion by BrushBenu68** [riffnshred/nhl-led-scoreboard#351] [General]
anyone else get an error when trying to dosudo chmod +x install.sh

**Comment by pjockey** [comment]
try `sudo chmod +x scripts/install.sh`

**Reply by BrushBenu68** [reply]
still says there is no such file or directory

**Comment by pjockey** [comment]
Are you in the  /nhl-led-scoreboard directory?

**Reply by BrushBenu68** [reply]
yes

**Reply by BrushBenu68** [reply]
idk what else to do

**Comment by pjockey** [comment]
Did you follow these steps exactly

```
git clone --recursive https://github.com/riffnshred/nhl-led-scoreboard
cd nhl-led-scoreboard/
chmod +x scripts/install.sh
./scripts/install.sh
```

**Reply by BrushBenu68** [reply]
i literally copied and pasted

**Reply by BrushBenu68** [reply]
are the last 2 lines 1 or 2 commands

**Comment by pjockey** [comment]
You can also start over and try again. Just do the command below to delete everything, then do the command in the post above.


```
cd
sudo rm -R nhl-led-scoreboard
```

How are you accessing the RPi?

**Reply by BrushBenu68** [reply]
ok ill try that now

**Reply by BrushBenu68** [reply]
now nothing happens when i run the chmod command

**Reply by BrushBenu68** [reply]
im accessing it from a hdmi from the pi to a monitor

**Reply by pjockey** [reply]
so you ran the command  sudo rm -R nhl-led-scoreboard?

**Reply by BrushBenu68** [reply]
yes and then i re did the other stuff and chmod command doesnt work

**Reply by pjockey** [reply]
hmmmm. do you have the bare minimum Pi OS? Or the graphical one?

**Reply by riffnshred** [reply]
The chmod command once executed does not give any feed back. All it does is changing user rights to be able to run the install.sh script. If you get nothing while running the chmod command provided, its normal.

**Reply by pjockey** [reply]
I was getting there!

**Comment by BrushBenu68** [comment]
omg its installing something

**Reply by BrushBenu68** [reply]
ok now when i run the test command there is no sych file or durectory for

**Reply by riffnshred** [reply]
Make sure you are in the nhl-led-scoreboard directory.

Run `cd` then `cd nhl-led-scoreboard` and run the test command again.

**Reply by BrushBenu68** [reply]
still didnt work

**Reply by BrushBenu68** [reply]
it says it cant open the file

**Reply by riffnshred** [reply]
Copy and paste the command you used and the error message here

**Reply by BrushBenu68** [reply]
ill drop a screen shot on second

**Reply by BrushBenu68** [reply]
![pi gekl](https://user-images.githubusercontent.com/95152079/143783385-80ea7352-3585-492a-9925-37c2ca8e55a6.jpg)

**Comment by riffnshred** [comment]
You are not running the right command. Make sure you use the commands from the step by step guide in the wiki section

**Reply by BrushBenu68** [reply]
i cannot find what i should be doing next, im so list

**Reply by riffnshred** [reply]
Are you following the howchoo tutorial by any chance???

**Reply by BrushBenu68** [reply]
yes...

**Reply by riffnshred** [reply]
Bingo !!! Im not associated with that website and their writers, and that tutorial is old and out of date. From now on, forget that tutorial and follow the instructions and information provided in this github repository. 

Start by reading the documentation in the main page [here](https://github.com/riffnshred/nhl-led-scoreboard). This is important in order to be able to run and troubleshoot your scoreboard. You will find two ways to install and run it. One is using a special raspberry pi OS image that is less complex but still requires patience. The other one is a manual installation that is a bit like what you have been doing in the Howchoo tutorial. Regardless of the way you are going to choose, it might not work the first time for many reasons. I even personally have to restart from the top of my own step by step guide because I entered the wrong option or ran the wrong command without realising. That is fine. just start from the top and once you do it, it will be rewarding. 

dont give up !!!

**Reply by BrushBenu68** [reply]
i tried using the image before but it gets stuck on the loading thing on the led panel

**Reply by BrushBenu68** [reply]
i just did it again and its still doing that

**Reply by riffnshred** [reply]
Which raspberry pi model are you using?

**Reply by BrushBenu68** [reply]
3 a+

**Reply by riffnshred** [reply]
ok thats good. Since you are using the img, you have a bunch of command you can use to troubleshoot.

https://github.com/falkyre/nhl-led-scoreboard-img#nhl-led-scoreboard-command-line-utilities

from the terminal, run this command:
`sb-issue`

 It will give you a link. copy and paste that link here.

**Reply by BrushBenu68** [reply]
https"//github.com/riffnshred/nhl-led-scoreboard/issues

i dont think its riight but thats alll it spits out

**Reply by riffnshred** [reply]
ok wait a bit we're double checking something.

**Reply by riffnshred** [reply]
when you run that command, you should see a link that looks like this

https://paste.ubuntu.com/p/some generated id/.  (worst case take a screenshot of it, we need the id)

That is the one you need. and instead of copy and pasting it here. create a new issue here

https://github.com/riffnshred/nhl-led-scoreboard/issues

Right now we are in the discussion section and that is not the right place to troubleshoot issues.
